api_v3/lib/types/KalturaUser.php

```diff
@@ -187,4 +194,12 @@ public function validateForUpdate($sourceObject, $propertiesToSkip = array())
 		$this->validateNames($sourceObject, self::$names);
 		parent::validateForUpdate($sourceObject, $propertiesToSkip);
 	}
+	
+	protected function validateSsoExclusion()
+	{
+		if(is_null($this->isSsoExcluded))
```

Comment:
dont think you will need this function if the default value will return as false from isSsoExcluded